src/coreclr-embedding-test/src/ScriptingManager.hpp

```diff
@@ -0,0 +1,45 @@
+//
+//  ScriptingManager.hpp
+//  embedding_test
+//
+//  Created by Leonidas Neftali Gonzalez Campos on 06/12/23.
```

Comment:
Loved the header part tho.

Should we create some scripts to create new headers and source files that include the header?

Reply:
Sure thing, I find that a good practice